doc/modules/model_evaluation.rst

```diff
@@ -231,6 +231,7 @@ Others also work in the multiclass case:
 .. autosummary::
    :template: function.rst
 
+   cohen_kappa_score
```

Comment:
@amueller kappa is used as a classification metric,
 see here https://www.kaggle.com/c/prudential-life-insurance-assessment/details/evaluation